aodntools/timeseries_products/velocity_hourly_timeseries.py

```diff
@@ -54,14 +54,14 @@ def get_resampled_values(nc_cell, ds, slice_start, varlist, binning_function, ep
     """
     df_cell = nc_cell[varlist].squeeze().to_dataframe()
     ## back the index 30min
-    df_cell.index = df_cell.index - pd.Timedelta(minutes=30)
+    df_cell.index = df_cell.index + pd.Timedelta(minutes=30)
     # TODO: shift timestamps to centre of sampling interval
```

Comment:
Comment not updated with code -  you are advancing the index instead.

Reply:
I imagine the code was wrong before, so the commit could highlight that

Reply:
It wasn't wrong, it just took two steps instead of one: it subtracted 30min, resampled, then _added an hour_.
So I simplified it to just adding 30min, which is exactly equivalent. I did a test comparison and all the variables in files generated with the two version of the code are exactly the same.

Reply:
comment changed to "shift the index forward 30min to centre the bins on the hour"